data-model/src/test/groovy/io/micronaut/data/model/PageSpec.groovy

```diff
@@ -122,11 +122,40 @@ class PageSpec extends Specification {
         def json = serdeMapper.writeValueAsString(pageable)
 
         then:
-        json == '{"size":3,"number":0,"sort":{}}'
+        json == '{"size":3,"number":0,"sort":{},"mode":"OFFSET"}'
         def deserializedPageable = serdeMapper.readValue(json, Pageable)
```

Comment:
We should support deserializing with not mode to support backward compatibility.